### PR TITLE
Ingo comments

### PR DESCRIPTION
**Pull request by jones1994**
### Description
Me an lisby went through the comments Ingo made on chapter 3 and 4 and fixed the ones which could immidiatly be fixed.

### Changelog
- Chapter 3 and 4

## FOR REVIEWER [Do not remove]
A small checklist of things to note when you are doing a review. Make sure these things apply before going into master.
 - Did the PR actually fix the issue
 - Comment anything that doesn't make sense or anything you want clarified. Many comments are always ok
 - Make sure this is ready for exams if this is going into master
 - It's always a good idea having another reviewer look at a PR as well, if it's major. get a buddy.

### Regarding filenames
- all .tex files use pascal case
- all other files [except in /code/] use snake case

### Regarding code [if applicable]
- Check [coding conventions](https://kotlinlang.org/docs/reference/coding-conventions.html)
- Are public methods documented? 
- Are there testcases for new features / methods?
- Have you tried running the source and tested things that cannot be test-cased?

### Regarding Report [if applicable]
- Does each line have a linechange in the sourcecode?
- Does sentences make sense?
- Did you do spellchecking?
- Citations?

### CODE REVIEW

File: report/3.Theory/LanguageSpecification.tex [hunk 1]
```diff
@@ -1,41 +1,43 @@
...
 \subsection{Types}
 HCL includes only three primitive types, numbers (\textbf{\textit{num}}), strings (\textbf{\textit{txt}}) and the boolean type (\textbf{\textit{bool}}).\\
 Numbers are represented using double-precision floating-point format, also known as the \textbf{\textit{double}} type.
 Booleans are represented using \textbf{True} or \textbf{False}.
 
-The collection types of HCL are \textbf{\textit{tuples}} and \textbf{\textit{lists}} where:\\
+The collection types of HCL are \textbf{\textit{tuples}} and \textbf{\textit{lists}} where, 
 \textbf{\textit{tuples}} are structures composed of multiple types and \textbf{\textit{lists}} are collections of a single type.
-Dictionaries, which are collections of key-value pairs, are deemed NTH and are not planned for implementation. 
+Dictionaries are a staple in modern languages, they are not deemed necessary for the scope of this "first specification" of HCL.
```
Comment (Lambcreek) [inline report/3.Theory/LanguageSpecification.tex]: Måske tilføj et "but" efter kommaet

File: report/3.Theory/LanguageSpecification.tex [hunk 2]
```diff
@@ -47,17 +49,17 @@ \subsection{Types}
...
 Most of the operators in HCL are defined as functions.
-This means that all arithmetic operators are infix-functions that take two numbers as parameters. 
-The only operators that remain operators are the assign-operator (\textbf{=}) and the encapsulation-operators, namely parenthesis, square-brackets and curly-brackets.
+This means that all arithmetic operators are infix-functions, which has expressions as parameters. 
+The only operators that remain operators are the assign-operator and the encapsulation-operators, namely parenthesis, square-brackets and curly-brackets.
 
-Not having operators for arithmetic operations also means that there are no native operator precedence in HCL.
-This can be amended by structuring calculations in a left-to-right manner, or by encapsulating operations with parentheses.
+Not having operators for arithmetic operations also means that there is no native operator precedence in HCL.
+This can be amended by structuring calculations in a left-to-right manner, or by encapsulating operations with parentheses, since parentheses has higher precedence than operators.
```
Comment (Lambcreek) [inline report/3.Theory/LanguageSpecification.tex]: "parentheses has higher precedence"
skal ændres til
"parentheses have higher precedence"

File: report/3.Theory/LanguageSpecification.tex [hunk 3]
```diff
@@ -47,17 +49,17 @@ \subsection{Types}
...
 
 \subsection{Functions}
 \textbf{Function Declaration}\\
 The type of a function's parameters, as well as its return type, are explicitly declared when a function is defined. 
 The declaration is structured as an assignment, where the functions name is declared on the left-hand-side, prefixed by either \textbf{\textit{var}} or \textbf{\textit{func}}. 
-The parameters of the function is declared on the right-hand-side inside parentheses, followed by a colon (\textbf{\textit{:}}), which declares the return type. 
+The parameters of the function is declared on the right-hand-side, inside parentheses, followed by a colon, which declares the return type.
```
Comment (Lambcreek) [inline report/3.Theory/LanguageSpecification.tex]: "The parameters of the function is"
skal ændres til
"The parameters of the function are"

File: report/3.Theory/LanguageSpecification.tex [hunk 4]
```diff
@@ -88,7 +90,9 @@ \subsection{Functions}
 
 \textbf{Function Calls}\\
 In order to better reflect the English language, function calls are structured in a way where the first parameter to a function is written before the function call itself.
+This first parameter  will be referred to as the prefix parameter.
```
Comment (C0DK) [inline report/3.Theory/LanguageSpecification.tex]: two spaces between "parameter  will"

File: report/3.Theory/ContextFreeGrammars.tex [hunk 5]
```diff
@@ -40,4 +40,4 @@ \section{Context-free Grammar of HCL}
 	\texttt{<Sentence>}\to & \texttt{ \{word\} \$}
 \end{align*}
 
-The grammar of HCL has been described using EBNF, and has been included in the appendix (Appendix \ref{AppendixEBNF}).
+The grammar of HCL is described using EBNF, is included in appendix \ref{AppendixEBNF}.
```
Comment (Abildgaard96) [inline report/3.Theory/ContextFreeGrammars.tex]: should be -> and is included

**Pull request merged**